### PR TITLE
Docs: Misc Minor Updates

### PR DESCRIPTION
**Pull request by dakanji**

### CODE REVIEW

File: Docs/Configuration.tex [hunk 1]
```diff
@@ -558,11 +558,11 @@ \subsection{Directory Structure}\label{directory-structure}
 
 \emph{Note}: It is not guaranteed that paths longer than
 \texttt{OC\_STORAGE\_SAFE\_PATH\_MAX} (128 characters including
-\texttt{\\0}-terminator) will be accessible within OpenCore.
+the null-terminator) will be accessible within OpenCore.
```
Comment (vit9696) [inline Docs/Configuration.tex:561]: It is not null terminator. Either nul, with one l, or \0. Better leave unchanged in my opinion.

**Comment by Andrey1970AppleLife** [comment]
Return FileVault!

File: Docs/Configuration.tex [hunk 2]
```diff
@@ -3108,7 +3098,7 @@ \subsection{Boot Properties}\label{miscbootprops}
     \emph{Note 1}: Apple's boot picker partially supports placing a volume icon file
     at the operating system's \texttt{Data} volume root (\texttt{/System/Volumes/Data/} when
     mounted at default location within macOS). This approach is broken: that file is not
-    accessible either by OpenCanopy or by Apple's own boot picker when FileVault 2 is
+    accessible either by OpenCanopy or by Apple's own boot picker when FaultVault 2 is
```
Comment (vit9696) [inline Docs/Configuration.tex:3101]: It is FileVault, not FaultVault.

File: Docs/Configuration.tex (hunk 1 above)
Comment (dakanji) [reply Docs/Configuration.tex:561]: Changed

Comment (vit9696) [reply Docs/Configuration.tex:561]: For me it still shows broken.

Comment (dakanji) [reply Docs/Configuration.tex:561]: Meant to be changed and pushed.

File: Docs/Configuration.tex (hunk 2 above)
Comment (dakanji) [reply Docs/Configuration.tex]: Actually thought I saw a commit fixing this earlier and rebased on current status.
Anyway, fixed now

File: Docs/Configuration.tex [hunk 3]
```diff
@@ -6665,8 +6654,8 @@ \subsection{Quirks Properties}\label{uefiquirkprops}
   This is a very rough workaround to circumvent the \texttt{Still\ waiting\ for\ root\ device} message
   on some APTIO IV firmware (ASUS Z87-Pro) particularly when using FileVault 2.
   It appears that for some reason, they execute code in parallel to \texttt{EXIT\_BOOT\_SERVICES},
-  which results in the SATA controller being inaccessible from macOS. A better approach should be
-  found in some future. Expect 3 to 5 seconds to be adequate when this quirk is needed.
+  which results in the SATA controller being inaccessible from macOS. A better approach would be
```
Comment (vit9696) [inline Docs/Configuration.tex:6657]: Actually no such intent. It is rather, if anybody knows a better way, report and help us implement it.

File: Docs/Configuration.tex [hunk 4]
```diff
@@ -6728,26 +6717,25 @@ \subsection{Quirks Properties}\label{uefiquirkprops}
...
 \item
   \texttt{UnblockFsConnect}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
   \textbf{Description}: Some types of firmware block partition handles by opening them
-  in \texttt{By\ Driver} mode, resulting in being unable to install File System protocols.
+  in \texttt{By\ Driver} mode, resulting in an inability to install File System protocols.
 
-  \emph{Note}: The quirk is mostly relevant for select HP laptops with no drives listed.
+  \emph{Note}: The quirk is mostly relevant for some HP laptops without listed drives.
```
Comment (vit9696) [inline Docs/Configuration.tex:6738]: This is a bit misworded even originally. Maybe smth similar to: It is relevant for some HP laptops with unsuccessful drive detection resulting in no boot entries.

**Review by vit9696 — COMMENTED** [review]
A few more :)

**Comment by vit9696** [comment]
Actually, in some places we have FileVault, and in others, File Vault. Need to check how Apple names it and fix consistency.

File: Docs/Configuration.tex (hunk 3 above)
Comment (dakanji) [reply Docs/Configuration.tex:6657]: How about ...
`A better approach is required and Acidanthera is open to suggestions.`

Comment (vit9696) [reply Docs/Configuration.tex:6657]: Fine to me.

File: Docs/Configuration.tex (hunk 4 above)
Comment (dakanji) [reply Docs/Configuration.tex:6738]: How about...
`This quirk is typically useful on some HP laptops where unsuccessful drive detection results in an absence of boot entries.`

**Comment by dakanji** [comment]
Apple uses `FileVault`
Example: https://support.apple.com/en-us/HT204837

File: Docs/Configuration.tex (hunk 4 above)
Comment (dakanji) [reply Docs/Configuration.tex:6738]: Is it even necessary to mention `HP laptops`?
This seems better: `This quirk is useful in cases where unsuccessful drive detection results in an absence of boot entries.`

**Pull request merged**